docs/symphony-book/get-started/deploy_solution_to_azure_iot_edge.md

```diff
@@ -73,20 +76,84 @@ You need to prepare a Linux virtual machine or physical device for IoT Edge. In
   sudo iotedge config mp --connection-string '<REPLACE_WITH_DEVICE_CONNECTION_STRING>'
   ```
 
+**Note**
+If you encounter below package dependency error during installing `aziot-edge`:
+```
+The following packages have unmet dependencies:
+ aziot-identity-service : Depends: libssl1.1 (>= 1.1.1) but it is not installable
+```
+
+This is because Ubuntu 22.04 uses `libssl3` and `libssl1.1` is deprecated on it. To address the issue, check the latest versions for Edge and IoT Identity Service and install them from the release page.
+
+```shell
+# check the latest aziot-edge version
+wget -qO- https://raw.githubusercontent.com/Azure/azure-iotedge/main/product-versions.json | jq -r '
+  .channels[]
+  | select(.name == "stable").products[]
+  | select(.id == "aziot-edge").components[]
+  | select(.name == "aziot-edge").version
+'
+# check the latest aziot-identity-service version
+wget -qO- https://raw.githubusercontent.com/Azure/azure-iotedge/main/product-versions.json | jq -r '
+  .channels[]
+  | select(.name == "stable").products[]
+  | select(.id == "aziot-edge").components[]
+  | select(.name == "aziot-identity-service").version
+'
+
+# download and install 
+wget https://github.com/Azure/azure-iotedge/releases/download/<AZIOT_EDGE_VERSION>/aziot-identity-service_<AZIOT_IDENTITY_SERVICE_VERSION>-1_ubuntu22.04_amd64.deb -O aziot-identity-service.deb
```

Comment:
1. Add definition for AZIOT_EDGE_VERSION and AZIOT_IDENTITY_SERVICE_VERSION
2. The first parameter should be AZIOT_IDENTITY_SERVICE_VERSION or AZIOT_EDGE_VERSION? 
3. maybe switch the order of aziot-identity-service.deb and aziot-edge.deb to keep consistency with the previous section.

Reply:
Added definitions and updated the sequence. The first parameter is `AZIOT_EDGE_VERSION`, they are under a same parent version.